Main.java

```diff
@@ -0,0 +1,43 @@
+package ru.geekbrains.java;
+
+public class Main {
+    // задание 1
+    public static void main(String[] args) {
+    }
```

Comment:
what to run?
```    public static void main(String[] args) {
    }

Reply:
Here it is needed to make an empty main method.